test/create-open.test.js

```diff
@@ -207,6 +207,36 @@ Object.keys(testAPIs).forEach(API => {
             assert.deepEqual(db.access.write, [orbitdb.identity.id])
           })
         })
+        describe('Meta', function() {
+          before(async () => {
+            if (db) {
+              await db.close()
+              await db.drop()
+            }
+          })
+
+          afterEach(async () => {
+            if (db) {
+              await db.close()
+              await db.drop()
+            }
+          })
+
+          it('creates a manifest with no meta field', async () => {
+            db = await orbitdb.create('no-meta', 'feed')
+            const manifest = await io.read(ipfs, db.address.root)
+            assert.strictEqual(db.options.meta, undefined)
+            assert.deepStrictEqual(Object.keys(manifest).filter(k => k === 'meta'), [])
+          })
+
+          it('creates a manifest with a meta field', async () => {
+            const meta = { test: 123 }
+            db = await orbitdb.create('meta', 'feed', { meta })
+            const manifest = await io.read(ipfs, db.address.root)
+            assert.deepStrictEqual(db.options.meta, meta)
+            assert.deepStrictEqual(Object.keys(manifest).filter(k => k === 'meta'), ['meta'])
```

Comment:
For similar reasons perhaps here we could use `assert.notEqual(manifest.meta, undefined)`?

Reply:
i guess either one works although using not notEqual here the next line, line 237, seems close to redundant. line 237's type of check is really only needed in the no-meta test, if you dont see a need for using deepStrictEqual here you could get rid of one of those lines.

Reply:
made a commit below that changes db.options.meta to manifest.meta in both tests. it seemed like it would solve the code convention issue here.